docs/conf/portland/2024/writing-day.rst

```diff
@@ -164,6 +164,19 @@ Our Writing Day goal is to onboard contributors and improve the Museum API defin
 
 The Museum API was built specifically to teach OpenAPI concepts and to create a better developer experience. The more realistic and complex the Museum API is, the better it can demonstrate all that OpenAPI has to offer.
 
+Write the Docs Documentation Salary Survey
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Project Organizer Kay Smoljak, she/her
+
+The WTD Salary Survey has been running annually since 2019, with the goal of identifying appropriate salary ranges and providing a basis for pay negotiations, as well as spark discussion on other topics such as remote work, pay transparency, and job satisfaction.   
```

Comment:
```suggestion
The WTD Salary Survey has been running annually since 2019, with the goal of identifying appropriate salary ranges and providing a basis for pay negotiations, as well as sparking discussion on other topics such as remote work, pay transparency, and job satisfaction.   
```